pyproject.toml

```diff
@@ -54,7 +54,7 @@ isa = [
 img2mml = ["fastapi", "uvicorn", "torch", "torchvision", "python-multipart"]
 
 # dependencies for text reading utilities.
-tr = ["transformers", "gensim", "SPARQLWrapper", "tqdm", "pandas"]
+tr = ["transformers", "SPARQLWrapper", "tqdm", "pandas", "askem_extractions[all]@git+https://github.com/ml4ai/ASKEM-TA1-DataModel"]
```

Comment:
Awesome!  Have you confirmed (ex. from the REPL) this pulls in your dep?

Reply:
Yes, it works. It builds and I am able to import the code from within a python session inside the container